firebase-vscode/src/options.ts

```diff
@@ -1,19 +1,20 @@
-import { Config as cliConfig } from "../../src/config";
 import { FirebaseConfig } from "../../src/firebaseConfig";
 import { FirebaseRC } from "../common/firebaserc";
 import { RC } from "../../src/rc";
 import { BaseOptions, Options } from "../../src/options";
 import { Command } from "../../src/command";
 import { ExtensionContext } from "vscode";
 import { setInquirerOptions } from "./stubs/inquirer-stub";
+import * as commandUtils from "../../src/emulator/commandUtils";
 
 /**
  * User-facing CLI options
  * Passed to command.prepare()
  */
 
-interface CliOptions extends Omit<BaseOptions, "config"> {
-  config: string;
+interface CliOptions extends BaseOptions {
```

Comment:
See the other comment about config but I think the thing is that `command.prepare` takes CLI options as passed in directly from user input, and then it does some renaming, and this `CliOptions` interface reflects the type that `command.prepare` takes, and in that case, `config` is a string.

Reply:
Hmm - I see what you're saying - this config though is overloaded to the point of being fragile. At some point the `options.config` represents the firebase.json compiled config right? How can we separate the two so as not to cross wires. Is that what command.prepare does, replace options.config with an object instead of a string?